baselayout/flatcar-profile.sh

```diff
@@ -32,7 +32,7 @@ if [[ $- == *i* ]]; then
 		)
 	fi
 
-	FAILED=$(systemctl list-units --state=failed --no-legend | tr '●' ' ')
+	FAILED=$(systemctl list-units --state=failed --no-legend | tr '×' ' ')
```

Comment:
The change looks good. I have one question though: would it make sense to have `tr '●×' ' '`? So we would handle both old and new symbols? That would come in handy if we need to make a change for lts/stable/beta too without creating a separate branch just for them?